### ISSUE TITLE
FETCH_COLUMN, FETCH_KEY_PAIR

### ISSUE THREAD

**Issue by hajcu** [phalcon/cphalcon#10138]
Fetch column, and fetch key pairs, not working.

This in phalcon 2.0 not work.
````
$result_set->setFetchMode(PDO::FETCH_COLUMN,0);
````

I think phalcon should recoginze this:
````
$sth->fetchAll(PDO::FETCH_COLUMN, 0);
````

But function fetchAll don't see any parameters :)

**Comment by salarmehr** [comment]
possible duplicate of https://github.com/phalcon/cphalcon/issues/3191

**Comment by andresgutierrez** [comment]
This is fixed in Phalcon 2.0.4
